Compiling fails when encountering new compiler warnings

Using

```
oqs = { version="0.7.2", default-features = false, features = ["kyber"] }
```

I get when building

```
   Compiling oqs-sys v0.7.2
error: failed to run custom build command for `oqs-sys v0.7.2`

Caused by:
  process didn't exit successfully: `/Users/bas/scm/rust-hpke/target/debug/build/oqs-sys-450355f50f2b9656/build-script-build` (exit status: 101)
  --- stdout
  CMAKE_TOOLCHAIN_FILE_x86_64-apple-darwin = None
  CMAKE_TOOLCHAIN_FILE_x86_64_apple_darwin = None
  HOST_CMAKE_TOOLCHAIN_FILE = None
  CMAKE_TOOLCHAIN_FILE = None
  CMAKE_GENERATOR_x86_64-apple-darwin = None
  CMAKE_GENERATOR_x86_64_apple_darwin = None
  HOST_CMAKE_GENERATOR = None
  CMAKE_GENERATOR = None
  CMAKE_PREFIX_PATH_x86_64-apple-darwin = None
  CMAKE_PREFIX_PATH_x86_64_apple_darwin = None
  HOST_CMAKE_PREFIX_PATH = None
  CMAKE_PREFIX_PATH = None
  CMAKE_x86_64-apple-darwin = None
  CMAKE_x86_64_apple_darwin = None
  HOST_CMAKE = None
  CMAKE = None
  running: "cmake" "/Users/bas/.cargo/registry/src/github.com-1ecc6299db9ec823/oqs-sys-0.7.2/liboqs" "-DCMAKE_OSX_ARCHITECTURES=x86_64" "-DOQS_BUILD_ONLY_LIB=Yes" "-DOQS_DIST_BUILD=Yes" "-DOQS_ENABLE_KEM_BIKE=No" "-DOQS_ENABLE_KEM_CLASSIC_MCELIECE=No" "-DOQS_ENABLE_KEM_FRODOKEM=No" "-DOQS_ENABLE_KEM_HQC=No" "-DOQS_ENABLE_KEM_KYBER=Yes" "-DOQS_ENABLE_KEM_NTRU=No" "-DOQS_ENABLE_KEM_NTRUPRIME=No" "-DOQS_ENABLE_KEM_SABER=No" "-DOQS_ENABLE_KEM_SIDH=No" "-DOQS_ENABLE_KEM_SIKE=No" "-DOQS_ENABLE_SIG_DILITHIUM=No" "-DOQS_ENABLE_SIG_FALCON=No" "-DOQS_ENABLE_SIG_PICNIC=No" "-DOQS_ENABLE_SIG_RAINBOW=No" "-DOQS_ENABLE_SIG_SPHINCS=No" "-DOQS_USE_OPENSSL=No" "-DCMAKE_INSTALL_PREFIX=/Users/bas/scm/rust-hpke/target/debug/build/oqs-sys-d30719a70bf74e8d/out" "-DCMAKE_C_FLAGS= -ffunction-sections -fdata-sections -fPIC -m64 -arch x86_64" "-DCMAKE_C_COMPILER=/usr/bin/cc" "-DCMAKE_CXX_FLAGS= -ffunction-sections -fdata-sections -fPIC -m64 -arch x86_64" "-DCMAKE_CXX_COMPILER=/usr/bin/c++" "-DCMAKE_ASM_FLAGS= -ffunction-sections -fdata-sections -fPIC -m64 -arch x86_64" "-DCMAKE_ASM_COMPILER=/usr/bin/cc" "-DCMAKE_BUILD_TYPE=Release"
  -- Configuring done (0.1s)
  -- Generating done (0.3s)
  -- Build files have been written to: /Users/bas/scm/rust-hpke/target/debug/build/oqs-sys-d30719a70bf74e8d/out/build
  running: "cmake" "--build" "." "--target" "oqs" "--config" "Release"
  [  0%] Built target xkcp_low_keccakp_1600_plain64
  [  1%] Built target xkcp_low_keccakp_1600_avx2
  [  1%] Building C object src/kem/kyber/CMakeFiles/kyber_1024_90s_avx2.dir/pqcrystals-kyber_kyber1024-90s_avx2/indcpa.c.o
  [  1%] Building C object src/common/CMakeFiles/common.dir/aes/aes.c.o
  [  2%] Built target xkcp_low_keccakp_1600times4_avx2
  [  3%] Built target xkcp_low_keccakp_1600times4_serial
  [  4%] Building C object src/kem/kyber/CMakeFiles/kyber_512_ref.dir/kem_kyber_512.c.o
  [  5%] Building ASM object src/kem/kyber/CMakeFiles/kyber_512_avx2.dir/pqcrystals-kyber_kyber512_avx2/basemul.S.o
  [  5%] Building C object src/kem/kyber/CMakeFiles/kyber_512_ref.dir/pqcrystals-kyber_kyber512_ref/cbd.c.o
  [  6%] Building C object src/kem/kyber/CMakeFiles/kyber_512_avx2.dir/pqcrystals-kyber_kyber512_avx2/cbd.c.o
  [  7%] Building C object src/kem/kyber/CMakeFiles/kyber_768_ref.dir/kem_kyber_768.c.o
  [  8%] Building ASM object src/kem/kyber/CMakeFiles/kyber_768_avx2.dir/pqcrystals-kyber_kyber768_avx2/basemul.S.o
  [  9%] Building C object src/common/CMakeFiles/common.dir/aes/aes_c.c.o
  [  9%] Building C object src/kem/kyber/CMakeFiles/kyber_1024_90s_avx2.dir/pqcrystals-kyber_kyber1024-90s_avx2/kem.c.o
  [ 10%] Building C object src/kem/kyber/CMakeFiles/kyber_768_avx2.dir/pqcrystals-kyber_kyber768_avx2/cbd.c.o
  [ 10%] Building C object src/common/CMakeFiles/common.dir/aes/aes128_ni.c.o
  [ 11%] Building C object src/common/CMakeFiles/common.dir/aes/aes256_ni.c.o
  [ 11%] Building C object src/kem/kyber/CMakeFiles/kyber_512_avx2.dir/pqcrystals-kyber_kyber512_avx2/consts.c.o
  [ 12%] Building C object src/kem/kyber/CMakeFiles/kyber_1024_90s_avx2.dir/pqcrystals-kyber_kyber1024-90s_avx2/poly.c.o
  [ 12%] Building C object src/kem/kyber/CMakeFiles/kyber_768_avx2.dir/pqcrystals-kyber_kyber768_avx2/consts.c.o
  [ 13%] Building C object src/kem/kyber/CMakeFiles/kyber_1024_90s_avx2.dir/pqcrystals-kyber_kyber1024-90s_avx2/rejsample.c.o
  [ 14%] Building ASM object src/kem/kyber/CMakeFiles/kyber_768_avx2.dir/pqcrystals-kyber_kyber768_avx2/fq.S.o
  [ 15%] Building C object src/common/CMakeFiles/common.dir/sha2/sha2.c.o
  [ 16%] Building ASM object src/kem/kyber/CMakeFiles/kyber_512_avx2.dir/pqcrystals-kyber_kyber512_avx2/fq.S.o
  [ 16%] Building C object src/kem/kyber/CMakeFiles/kyber_768_avx2.dir/pqcrystals-kyber_kyber768_avx2/indcpa.c.o
  [ 16%] Building C object src/common/CMakeFiles/common.dir/sha2/sha2_c.c.o
  [ 16%] Building C object src/kem/kyber/CMakeFiles/kyber_512_avx2.dir/pqcrystals-kyber_kyber512_avx2/indcpa.c.o
  [ 17%] Building ASM object src/kem/kyber/CMakeFiles/kyber_768_avx2.dir/pqcrystals-kyber_kyber768_avx2/invntt.S.o
  [ 18%] Building C object src/common/CMakeFiles/common.dir/sha3/xkcp_sha3.c.o
  [ 19%] Building ASM object src/kem/kyber/CMakeFiles/kyber_512_avx2.dir/pqcrystals-kyber_kyber512_avx2/invntt.S.o
  [ 19%] Building C object src/common/CMakeFiles/common.dir/sha3/xkcp_sha3x4.c.o
  [ 20%] Building C object src/kem/kyber/CMakeFiles/kyber_768_avx2.dir/pqcrystals-kyber_kyber768_avx2/kem.c.o
  [ 21%] Building C object src/common/CMakeFiles/common.dir/common.c.o
  [ 27%] Built target kyber_1024_90s_avx2
  [ 28%] Building C object src/kem/kyber/CMakeFiles/kyber_512_avx2.dir/pqcrystals-kyber_kyber512_avx2/kem.c.o
  [ 28%] Building ASM object src/kem/kyber/CMakeFiles/kyber_768_avx2.dir/pqcrystals-kyber_kyber768_avx2/ntt.S.o
  [ 29%] Building C object src/kem/kyber/CMakeFiles/kyber_768_avx2.dir/pqcrystals-kyber_kyber768_avx2/poly.c.o
  [ 30%] Building C object src/kem/kyber/CMakeFiles/kyber_768_avx2.dir/pqcrystals-kyber_kyber768_avx2/polyvec.c.o
  [ 30%] Building ASM object src/kem/kyber/CMakeFiles/kyber_512_avx2.dir/pqcrystals-kyber_kyber512_avx2/ntt.S.o
  [ 31%] Building C object src/common/CMakeFiles/common.dir/pqclean_shims/nistseedexpander.c.o
  [ 31%] Building C object src/kem/kyber/CMakeFiles/kyber_768_avx2.dir/pqcrystals-kyber_kyber768_avx2/rejsample.c.o
  [ 31%] Building C object src/common/CMakeFiles/common.dir/pqclean_shims/fips202.c.o
  [ 32%] Building ASM object src/kem/kyber/CMakeFiles/kyber_768_avx2.dir/pqcrystals-kyber_kyber768_avx2/shuffle.S.o
  [ 33%] Building C object src/kem/kyber/CMakeFiles/kyber_512_avx2.dir/pqcrystals-kyber_kyber512_avx2/poly.c.o
  [ 34%] Building C object src/common/CMakeFiles/common.dir/pqclean_shims/fips202x4.c.o
  [ 35%] Building C object src/common/CMakeFiles/common.dir/rand/rand.c.o
  [ 36%] Building C object src/kem/kyber/CMakeFiles/kyber_512_avx2.dir/pqcrystals-kyber_kyber512_avx2/polyvec.c.o
  [ 36%] Building C object src/kem/kyber/CMakeFiles/kyber_768_avx2.dir/pqcrystals-kyber_kyber768_avx2/symmetric-shake.c.o
  [ 36%] Building C object src/common/CMakeFiles/common.dir/rand/rand_nist.c.o
  [ 36%] Building C object src/kem/kyber/CMakeFiles/kyber_512_avx2.dir/pqcrystals-kyber_kyber512_avx2/rejsample.c.o
  [ 37%] Building C object src/kem/kyber/CMakeFiles/kyber_768_avx2.dir/pqcrystals-kyber_kyber768_avx2/verify.c.o
  [ 38%] Building ASM object src/kem/kyber/CMakeFiles/kyber_512_avx2.dir/pqcrystals-kyber_kyber512_avx2/shuffle.S.o
  [ 38%] Building C object src/kem/kyber/CMakeFiles/kyber_512_avx2.dir/pqcrystals-kyber_kyber512_avx2/symmetric-shake.c.o
  [ 39%] Building C object src/kem/kyber/CMakeFiles/kyber_512_avx2.dir/pqcrystals-kyber_kyber512_avx2/verify.c.o
  [ 39%] Built target kyber_768_avx2
  [ 39%] Built target kyber_512_avx2
  [ 39%] Built target common

  --- stderr
  /Users/bas/.cargo/registry/src/github.com-1ecc6299db9ec823/oqs-sys-0.7.2/liboqs/src/kem/kyber/kem_kyber_512.c:9:31: error: a function declaration without a prototype is deprecated in all versions of C [-Werror,-Wstrict-prototypes]
  OQS_KEM *OQS_KEM_kyber_512_new() {
                                ^
                                 void
  1 error generated.
  make[3]: *** [src/kem/kyber/CMakeFiles/kyber_512_ref.dir/kem_kyber_512.c.o] Error 1
  make[3]: *** Waiting for unfinished jobs....
  /Users/bas/.cargo/registry/src/github.com-1ecc6299db9ec823/oqs-sys-0.7.2/liboqs/src/kem/kyber/kem_kyber_768.c:9:31: error: a function declaration without a prototype is deprecated in all versions of C [-Werror,-Wstrict-prototypes]
  OQS_KEM *OQS_KEM_kyber_768_new() {
                                ^
                                 void
  1 error generated.
  make[3]: *** [src/kem/kyber/CMakeFiles/kyber_768_ref.dir/kem_kyber_768.c.o] Error 1
  make[2]: *** [src/kem/kyber/CMakeFiles/kyber_768_ref.dir/all] Error 2
  make[2]: *** Waiting for unfinished jobs....
  make[2]: *** [src/kem/kyber/CMakeFiles/kyber_512_ref.dir/all] Error 2
  make[1]: *** [src/CMakeFiles/oqs.dir/rule] Error 2
  make: *** [oqs] Error 2
  thread 'main' panicked at '
  command did not execute successfully, got: exit status: 2

  build script failed, must exit now', /Users/bas/.cargo/registry/src/github.com-1ecc6299db9ec823/cmake-0.1.50/src/lib.rs:1098:5
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Rust version:

```
stable-x86_64-apple-darwin (default)
rustc 1.67.1 (d5a82bbd2 2023-02-07)
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -15,8 +15,12 @@ bindgen = "0.65"
 autotools = "0.2"
 build-target = "0.4.0"
 
-[dependencies]
-oqs-sys = { version = "0.7.2", default-features = false, features = ["kems", "sigs"], optional = true}
+[dependencies.oqs-sys]
+git = "https://github.com/open-quantum-safe/liboqs-rust"
+commit = "f7bd5038018f2d69c55b642fcc373ae445ac378a"
+default-features = false
+features = ["kems", "sigs"]
+optional = true
 
 [features]
 default = []

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -11,7 +11,7 @@ keywords = ["wolfssl", "vpn", "lightway", "post-quantum", "cryptography"]
 links = "wolfssl"
 
 [build-dependencies]
-bindgen = "0.64"
+bindgen = "0.65"
 autotools = "0.2"
 build-target = "0.4.0"
 

--- a/Earthfile
+++ b/Earthfile
@@ -1,5 +1,5 @@
 VERSION 0.6
-FROM rust:1.67.1
+FROM rust:1.69
 
 WORKDIR /wolfssl-sys
 

--- a/Earthfile
+++ b/Earthfile
@@ -6,6 +6,7 @@ WORKDIR /wolfssl-sys
 build-deps:
     RUN apt-get update -qqy
     RUN apt-get install -qqy autoconf autotools-dev libtool-bin clang cmake
+    RUN apt-get -y install --no-install-recommends bsdmainutils
     RUN rustup component add rustfmt
 
 copy-src:

--- a/Earthfile
+++ b/Earthfile
@@ -4,8 +4,8 @@ FROM rust:1.67.1
 WORKDIR /wolfssl-sys
 
 build-deps:
-    RUN apt-get update -qqy
-    RUN apt-get install -qqy autoconf autotools-dev libtool-bin clang cmake
+    RUN apt-get update -qq
+    RUN apt-get install --no-install-recommends -qq autoconf autotools-dev libtool-bin clang cmake
     RUN apt-get -y install --no-install-recommends bsdmainutils
     RUN rustup component add rustfmt
 

--- a/build.rs
+++ b/build.rs
@@ -160,7 +160,7 @@ fn main() -> std::io::Result<()> {
         .header("wrapper.h")
         .clang_arg(format!("-I{dst_include}/"))
         .parse_callbacks(Box::new(ignored_macros))
-        .rustfmt_bindings(true);
+        .formatter(bindgen::Formatter::Rustfmt);
 
     let builder = builder
         .allowlist_file(format!("{dst_include}/wolfssl/.*.h"))
@@ -170,8 +170,6 @@ fn main() -> std::io::Result<()> {
     let builder = builder.blocklist_function("wolfSSL_BIO_vprintf");
 
     let bindings: bindgen::Bindings = builder.generate().expect("Unable to generate bindings");
-
-    bindings.emit_warnings();
 
     // Write out the bindings
     bindings

